index.html

```diff
@@ -134,12 +134,15 @@ <h3 class="text-center">
                 </div>
             </div>
             <div class="form-group row">
-                <label for="Author" class="col-sm-2">Read :</label>
-                <div class="col-sm-1">
-                    <input type="checkbox" id="read">Yes</input>
-                    <input type="checkbox" id="read">No</input>
+                <label for="Favorite" class="col-sm-2 col-form-label">Read:</label>
```

Comment:
Putting the "favorite" and "read" switch button side by side makes it more accessible.

Reply:
Done